ICT4DNews/app/src/main/res/menu/ict4dnews_menu.xml

```diff
@@ -7,13 +7,13 @@
         android:icon="@drawable/ic_search_white_24dp"
         android:title="@string/search"
         app:actionViewClass="android.support.v7.widget.SearchView"
-        app:showAsAction="collapseActionView|ifRoom" />
```

Comment:
Please always use `ifRoom` and never `always`, even Android Studio shows a warning 😉

Reply:
it was `ifRoom,` but while using search when you change the orientation and close the search the items with `ifRoom` attribute are not visible anymore until the change of orientation again

Reply:
I cannot reproduce this? Works on my side? Can you test that again or provide me a detailed step-by-step guide to reproduce the bug?

Reply:
OK, please change it back to `always` but add a good comment for our future selves 😉